### ISSUE TITLE
Copy elements between reports: triggers

### ISSUE THREAD

**Issue by drjakob** [roschaefer/story.board#254]
As a reporter
I want to copy multiple trigger from one report to another report
in order to build new reports based on existing versions.
(analog to text.components in #253)

**Comment by roschaefer** [comment]
@drjakob I guess this and #253 is highly confusing from the user perspective. Are you sure you want this? How about just copying a report with all text components and triggers and then work on the two separately?

**Comment by drjakob** [comment]
Agree, should not be too confusing. But we are working on three reports at the same time. And new triggers are important for all of them. Probably we can just add a "duplicate"-trigger function? (Because then you can change the triggers report in trigger view –> edit by hand.)
